modules/markup/html_node.go

```diff
@@ -27,6 +35,32 @@ func processNodeAttrID(node *html.Node) {
 	}
 }
 
+func processFootnoteID(ctx *RenderContext, node *html.Node) {
+	for idx, attr := range node.Attr {
+		if attr.Key == "id" {
+			if !isAnchorIDFootnote(attr.Val) {
+				return
```

Comment:
continue?

Reply:
In a normal case, each attribute is only allowed once, so I think returning it directly is fine.